spark/hbase-spark/src/test/scala/org/apache/hadoop/hbase/spark/TableOutputFormatSuite.scala

```diff
@@ -31,14 +30,16 @@ import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, FunSuite}
 
 import scala.util.{Failure, Success, Try}
 
-
 // Unit tests for HBASE-20521: change get configuration(TableOutputFormat.conf) object first sequence from jobContext to getConf
 // this suite contains two tests, one for normal case(getConf return null, use jobContext), create new TableOutputformat object without init TableOutputFormat.conf object,
 // configuration object inside checkOutputSpecs came from jobContext.
 // The other one(getConf return conf object) we manually call "setConf" to init TableOutputFormat.conf, for making it more straight forward, we specify a nonexistent table
 // name in conf object, checkOutputSpecs will then throw TableNotFoundException exception
-class TableOutputFormatSuite extends FunSuite with
-  BeforeAndAfterEach with BeforeAndAfterAll with Logging{
+class TableOutputFormatSuite
+    extends FunSuite
+    with BeforeAndAfterEach
+    with BeforeAndAfterAll
+    with Logging {
```

Comment:
while here they are multiple lines...

Reply:
don't understand the style spotless applies, seems doesn't align

Reply:
Yes this one has 101 chars